tools/autograd/derivatives.yaml

```diff
@@ -415,8 +415,9 @@
 - name: deg2rad(Tensor self) -> Tensor
   self: deg2rad_backward(grad)
 
-- name: linalg_det(Tensor self) -> Tensor
-  self: linalg_det_backward(grad, self, result)
+- name: _det_lu_based_helper(Tensor self) -> (Tensor det, Tensor p, Tensor l, Tensor u)
+  self: _det_lu_based_helper_backward(grad, self, det, p, l, u)
+  output_differentiability: [True, False, False, False]
```

Comment:
No differentiability wrt `L` or `U`?

Reply:
No need, designed to be used inside det only. Also, makes it easier to handle in double backward.